src/main/java/CodeCheck/Main.java

```diff
@@ -3,7 +3,7 @@
 public class Main {
     public static void main(String[] args) throws Exception {
         Log.log("Starting up...");
-        Log.log("Running on %s.", "\n");
+        Log.log("Running on %s."); //Log.log("Running on %s.%n", System.getProperty("os.name"));
```

Comment:
revert?

Reply:
tror du gjorde det

Reply:
Yes, it was removed in a later commit. :)